host/xtest/include/ffa.h

```diff
@@ -0,0 +1,175 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
Instead of copying this file into this git we could make it available in the dev-kit by adding it to the list `incfiles-extra-host` in `ta/ta.mk`.

Reply:
Ack, [OP-TEE OS PR](https://github.com/OP-TEE/optee_os/pull/5522)